### DISCUSSION TITLE
Logstash Failing to Start dkms_autoinstaller

### DISCUSSION THREAD

**Discussion by lcooperAB** [Security-Onion-Solutions/securityonion#2287] [Unsupported Versions] [answered]
Weird issue after updating yesterday. Logstash is unable to start it's giving this weird error message:

`so-logstash: insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
insserv: Script dkms_autoinstaller is broken: incomplete LSB comment.
insserv: missing `Required-Stop:'  entry: please add even if empty.
`

So I went in and checked var/logs/logstash and it came up with this strange WARN code:

`[2020-12-08T21:30:25,075][WARN ][logstash.outputs.redis   ] Failed to send event to Redis {:event=>#<LogStash::Event:0x1906dcd0>, :identity=>"redis://@172.18.0.1:6379/0 list:logstash:redis", :exception=>#<$
`

Then it error'd out and shut down. I've checked usr/lib/dkms/dksm_autoinstaller and the line it says is incomplete looks accurate but I could be wrong. Here is the readout from that (comment tags removed to avoid Markdown):

!/bin/sh

 dkms_autoinstaller        A service to automatically install DKMS modules
                           for new kernels.
 chkconfig: 345 04 04
 description: An autoinstaller bootup service for DKMS

 BEGIN INIT INFO
 Provides: dkms_autoinstaller dkms
 Default-Start: 2 3 4 5
 Default-Stop:
 Required-Start: $local_fs
 Required-Stop:  $null
 Short-Description: Automatically install DKMS modules for new kernels
 Description: A service to automatically install DKMS modules for new kernels.
 END INIT INFO

So not real sure where to go to from here.  Might just speed up my timeline to upgrade to Security Onion 2. 

Any help would be appreciated!

**Comment by dougburks** [comment, marked as answer]
If you have questions or problems relating to the old Security Onion 16.04, then those should go to the old 16.04 Google Group as described here:
https://blog.securityonion.net/2020/10/community-support-forum-changes-for.html

Thanks!

**Reply by lcooperAB** [reply]
Thanks for that!
